api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/ObservationTranslatorImplTest.java

```diff
@@ -41,6 +41,7 @@
 import org.openmrs.module.fhir2.api.translators.EncounterReferenceTranslator;
```

Comment:
We'll need to add a test to this class as well to ensure that it's invoking the `referenceRangeTranslator`. I think this can be modeled on the test for `ObservationInterpretationTranslator`

Reply:
@ibacher I have written this test please suggest edits
```java
@Test
	public void toFhirResource_shouldInvokereferenceRangeTranslator() {
		ConceptNumeric conceptNumeric = new ConceptNumeric();
		conceptNumeric.setUuid(CONCEPT_UUID);
		
		List<Observation.ObservationReferenceRangeComponent> observationReferenceRangeComponentList = referenceRangeTranslator
		        .toFhirResource(conceptNumeric);
		
		assertThat(observationReferenceRangeComponentList, notNullValue());
	}
```

Reply:
So what else should we expect about this list, other than its not just a null value? Should it have any `Observation.ObservationReferenceRangeComponent`s or should it be empty?

Reply:
Also, feel free to keep pushing changes to this branch. You only need to squash everything down to one commit before it's merged.

Reply:
@ibacher Yes it should have items in it, so maybe something similar to what we did in `ObservationReferenceRangeTranslatorImplTest`

``` java
assertThat(result, notNullValue());
		assertThat(result, hasItem(hasProperty("low", hasProperty("value", equalTo(BigDecimal.valueOf(LOW_NORMAL_VALUE))))));
		assertThat(result,
		    hasItem(hasProperty("high", hasProperty("value", equalTo(BigDecimal.valueOf(HIGH_NORMAL_VALUE))))));
		
		assertThat(result,
		    hasItem(allOf(hasProperty("low", hasProperty("value", equalTo(BigDecimal.valueOf(LOW_NORMAL_VALUE)))),
		        hasProperty("type",
		            hasProperty("coding",
		                hasItem(allOf(
		                    hasProperty("system", equalTo("http://terminology.hl7.org/CodeSystem/referencerange-meaning")),
		                    hasProperty("code", equalTo("normal")))))))));
```